src/components/App/App.js

```diff
@@ -17,10 +17,15 @@ class App extends React.Component {
   }
 
   setUser = (newUser) => {
-    console.log(newUser);
     this.setState({ user: newUser });
   };
 
+  // dave thinking....maybe add the listings fetch nested under the areas in componentDidMount ? Would this be better than using componentDidUpdate to call once the area is selected like we talked about??
+  // then in the render have a listings container that gets routed to. <listingsContainer path=''> ?
+  // then would we store the current area here? but then you would have to also store the current listing and the favorites...is that too much? It does reduce the need for more class base components and keeps the data centralized. It also follows our wireframe. Maybe let's try it...
+  // this.state will probably need: user, areas, currentArea/currentListings(in order to pass down to the listingsContainer to display the correct listings for each area), selectedListing(for all the details), favorites. maybe an allListings array...i'm thinking this would be the easiest way to fetch and then store the data...otherwise we could add a key to the areas objects that holds all the listings but that might get tricky. allListings could then be iterated over to match area_ids in order to display the correct ones for each area? 
+  // BE CAREFUL NOT TO REPLICATE DATA IN this.state...we can do things like store ideas for favorites instead of storing the whole object. Then iterate over and match ids to display favorites.
+
```

Comment:
I'm still not sure the best way to fetch the listings data. It does make sense to do all the fetching in one place and keep the data centralized but not sure if we want to fetch everything on sign in if we don't need it. I wonder if it would make sense to have the fetch triggered by clicking on the View Listings button, but that would mean some wait time for the user.